SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, submit the issue via Github in the subsystem it applies to.
+
+For the OSAL subsystem, the submit the issue in the OSAL repository at https://github.com/nasa/osal/issues/new/choose.
+
+If you are not sure what subsystem it belongs to, then feel free to submit it at the general CFS level at https://github.com/nasa/cfs/issues/new/choose.
+
+In either case please use the "Bug Report" template and provide as much information as possible. Ensure to apply appropraite labels for each report. For security related reports, tag the issue with the "security" label.
+
+## Additional Support
+
+For additional support, reach out to cfs-community@lists.nasa.gov. This mailing list includes all community members/users of the NASA core Flight Software (cFS) product line.
```

Comment:
Replace "reach out" with "email us at"

Swap `cfs-community@lists.nasa.gov` with `cfs-program@lists.nasa.gov` since the community list will bounce messages from non-members. 

Keep the community mailing list sentence and add the clause  "For help using OSAL and cFS"

Reply:
@astrogeco Please let me know if the changes meets your requests. I will then apply the same format to cFS and other submodules.